google-cloud-spanner/lib/google/cloud/spanner/project.rb

```diff
@@ -217,14 +221,41 @@ def instance instance_id
         #     instance = job.instance
         #   end
         #
+        # @example Create instance using processsing units
+        #   require "google/cloud/spanner"
+        #
+        #   spanner = Google::Cloud::Spanner.new
+        #
+        #   job = spanner.create_instance "my-new-instance",
+        #                                 name: "My New Instance",
+        #                                 config: "regional-us-central1",
+        #                                 processing_units: 500,
+        #                                 labels: { production: :env }
+        #
+        #   job.done? #=> false
+        #   job.reload! # API call
+        #   job.done? #=> true
+        #
+        #   if job.error?
+        #     status = job.error
+        #   else
+        #     instance = job.instance
+        #   end
+        #
         def create_instance instance_id, name: nil, config: nil, nodes: nil,
-                            labels: nil
+                            processing_units: nil, labels: nil
           config = config.path if config.respond_to? :path
+
+          if nodes && processing_units
```

Comment:
I'd like to avoid adding too many logic in the admin client, can we leave this check to the backend? So we don't need to add this check and its test as well.

Reply:
done